services/src/main/java/io/druid/cli/validate/DruidJsonValidator.java

```diff
@@ -73,7 +86,23 @@ public void run()
       System.out.printf("File[%s] does not exist.%n", file);
     }
 
-    final ObjectMapper jsonMapper = new DefaultObjectMapper();
+    final Injector injector = GuiceInjectors.makeStartupInjector();
```

Comment:
Did the same thing and the second error message shown above was the result. I really have no idea on how it(guice?) works.